src/priyanka/Assignment30/FindMissing.java

```diff
@@ -0,0 +1,35 @@
+package priyanka.Assignment30;
+
+import java.util.Arrays;
+
+/*Assignment 30 : 29th Nov 2020 
+Program 1:  Find the missing number in integer array of 1 to 10?
+Hint: You have given an integer array which contains numbers from 1 to 10 but exactly one number is missing, you need to write a Java program to find that missing number in an array.
+*/
+
+public class FindMissing {
+	void findMissingNumber(int[] arr1, int[] arr2) {
+
+		for (int i = 0; i < arr1.length; i++) {
```

Comment:
You have to find missing number from single array

Reply:
Hi Harsh,  

updated the logic for missing number